tests/ci/ut_install.sh

```diff
@@ -22,12 +22,17 @@ curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/insta
 sudo service postgresql stop || echo no postgresql need to be stopped
 sleep 2
 
+# support for multi-architecture compilation
```

Comment:
If you are building an arm image, the replacement tag should have a tag with an arm suffix

Reply:
Cant this be bundled tho?
![image](https://user-images.githubusercontent.com/5084445/214855606-6c2d3a8d-2c58-4804-9d00-8ba35775655e.png)